.github/workflows/winget-submission.yml

```diff
@@ -0,0 +1,23 @@
+name: Submit package to Windows Package Manager Community Repository 
```

Comment:
Could just be called `submit-packages.yml` and `"Submit packages"` so we can use it for other package managers in the future